app/code/Magento/MediaGalleryCatalogIntegration/Test/Mftf/Test/AdminUploadSameImageDeleteFromTemporaryFolderTest.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminUploadSameImageDeleteFromTemporaryFolderTest">
+        <annotations>
+            <features value="AdminUploadSameImageDeleteFromTemporaryFolderTest"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/1792"/>
+            <title value="Image is deleted from tmp folder if is uploaded second time"/>
+            <description value="Image is deleted from tmp folder if is uploaded second time"/>
```

Comment:
@chalov-anton can you please provide a test case id for this scenario?

Reply:
https://studio.cucumber.io/projects/131313/test-plan/folders/943908/scenarios/4836631